<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LimsAuto</class>
 <widget class="QMainWindow" name="LimsAuto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lims Auto (1.0v)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="RPA_RUN">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>740</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RPA_STOP">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>740</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>파일</string>
    </property>
    <addaction name="menu_file_open"/>
    <addaction name="menu_file_Save"/>
    <addaction name="menu_file_SaveAs"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="menu_file_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_setting">
    <property name="title">
     <string>설정</string>
    </property>
    <addaction name="menu_setting_setting"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>도움말</string>
    </property>
    <addaction name="menu_help_help"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_setting"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="menu_file_open">
   <property name="text">
    <string>불러오기</string>
   </property>
  </action>
  <action name="menu_file_Save">
   <property name="text">
    <string>저장</string>
   </property>
  </action>
  <action name="menu_file_SaveAs">
   <property name="text">
    <string>다른이름으로 저장</string>
   </property>
  </action>
  <action name="menu_setting_setting">
   <property name="text">
    <string>설정</string>
   </property>
  </action>
  <action name="menu_help_help">
   <property name="text">
    <string>도움말 보기</string>
   </property>
  </action>
  <action name="menu_file_Quit">
   <property name="text">
    <string>끝내기</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
